pkg/ottl/README.md

```diff
@@ -11,305 +11,35 @@
 
 The OpenTelemetry Transformation Language is a language for transforming open telemetry data based on the [OpenTelemetry Collector Processing Exploration](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/processing.md).
 
-This package reads in OTTL statements and converts them to invokable Booleans and functions based on the OTTL's grammar.
+This package reads in OTTL statements and converts them to invokable functions/booleans based on the OTTL's grammar.
 
-The OTTL is signal agnostic; it is not aware of the type of telemetry on which it will operate.  Instead, the Booleans and functions returned by the package must be passed a `TransformContext`, which provide access to the signal's telemetry. Telemetry data can be accessed and updated through [Getters and Setters](#getters-and-setters).
+The OTTL is signal agnostic; it is not aware of the type of telemetry on which it will operate.  Instead, the functions/booleans returned by the package must be passed a `TransformContext`, which provide access to the signal's telemetry.
```

Comment:
I think we should move these lines into the language document or remove them. I'd like to keep this readme as user-facing as possible since this document will probably be frequented by users who are using OTTL in an existing processor.

Reply:
I looked for a good spot to put it in language.md and its really all covered throughout the file.  Opted to remove these statements instead.